src/libraries/System.Runtime.Caching/src/PACKAGE.md

```diff
@@ -1,37 +1,52 @@
 ## About
 
-[System.Runtime.Caching](https://learn.microsoft.com/en-us/dotnet/api/system.runtime.caching?view=dotnet-plat-ext-7.0) ([NuGet package](https://www.nuget.org/packages/System.Runtime.Caching/)) is a packaged set of simple caching API's derived from those of the same namespace available in .Net Framework since 4.0. This package is intended for use as a bridge when porting .Net Framework applications to .Net Core.
+<!-- A description of the package and where one can find more documentation -->
 
-This `System.Runtime.Caching` package can be used with any [.NET implementation](/dotnet/standard/net-standard#net-implementation-support) that targets .NET Standard 2.0 or later. For example:
-* .NET Core 3.1 or later.
-* .Net Framework 4.5 or later.
-* .Net 5.0 or late
+Packaged set of simple caching API's derived from those of the same namespace available in .NET Framework since 4.0. This package is intended for use as a bridge when porting .NET Framework applications to .NET.
 
 [Microsoft.Extensions.Caching.Memory](https://www.nuget.org/packages/Microsoft.Extensions.Caching.Memory/)/[IMemoryCache](https://learn.microsoft.com/en-us/aspnet/core/performance/caching/memory?view=aspnetcore-7.0) is recommended over `System.Runtime.Caching`/`MemoryCache` because it's better integrated into ASP.NET Core. For example, `IMemoryCache` works natively with ASP.NET Core [dependency injection](https://learn.microsoft.com/en-us/aspnet/core/fundamentals/dependency-injection?view=aspnetcore-7.0).
 
 > > [!IMPORTANT]
```

Comment:
Unrelated to you changes, but looks like NuGet markdown does not support [!IMPORTANT] blocks. I tried to preview it on NuGet gallery, and it renders like this:

![image](https://github.com/dotnet/runtime/assets/35516665/aa289957-5a42-448a-9e4f-1ff62b4022d1)

Reply:
Yes, we don't support [!IMPORTANT].  is this common use case for dotnet package? free feel to check issue on https://github.com/NuGet/NuGetGallery. Please let me know, I could create tracking isue on NuGet gallery repo. Thanks

Reply:
What would you recommend to use if we want to show something as important? Here is an example of how it looks in Microsoft Learn, but it doesn't necessarily need to look exactly like this: https://learn.microsoft.com/en-us/dotnet/api/system.xml.schema?view=net-7.0#remarks